sdk/face/azure-ai-vision-face/README.md

```diff
@@ -0,0 +1,484 @@
+
+face_client = FaceClient(endpoint, credential)
+```
+
+## Key concepts
+
+### FaceClient
+
+`FaceClient` provides operations for:
+
+ - Face detection and analysis: Detect human faces in an image and return the rectangle coordinates of their locations,
+   and optionally with landmarks, and face-related attributes. This operation is required as a first step in all the
+   other face recognition scenarios.
+ - Face recognition: Confirm that a user is who they claim to be based on how closely their face data matches the target face.
+   It includes Face verification ("one-to-one" matching) and Face identification ("one-to-many" matching).
+ - Finding similar faces from a smaller set of faces that look similar to the target face.
+ - Grouping faces into several smaller groups based on similarity.
+
+### FaceAdministrationClient
+
+`FaceAdministrationClient` is provided to interact with the following data structures that hold data on faces and
+persons for Face recognition:
+
+ - PersonDirectory
+ - FaceList
+ - LargeFaceList
+ - PersonGroup
+ - LargePersonGroup
+
+### FaceSessionClient
+
+`FaceSessionClient` is provided to interact with sessions which is used for Liveness detection.
+
+ - Create, query, and delete the session.
+ - Query the liveness and verification result.
+ - Query the audit result.
+
+### Long-running operations
+
+Long-running operations are operations which consist of an initial request sent to the service to start an operation,
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
+succeeded, to get the result.
+
+Methods that train a group (LargeFaceList, PersonGroup or LargePersonGroup), create/delete a Person/DynamicPersonGroup, 
+add a face or delete a face from a Person are modeled as long-running operations.
+The client exposes a `begin_<method-name>` method that returns an `LROPoller` or `AsyncLROPoller`. Callers should wait
+for the operation to complete by calling `result()` on the poller object returned from the `begin_<method-name>` method.
+Sample code snippets are provided to illustrate using long-running operations [below](#examples "Examples").
+
+## Examples
+
+The following section provides several code snippets covering some of the most common Face tasks, including:
+
+* [Detecting faces in an image](#face-detection "Face Detection")
+* [Identifying the specific face from a LargePersonGroup](#face-recognition-from-largepersongroup "Face Recognition from LargePersonGroup")
+* [Determining if a face in an video is real (live) or fake (spoof)](#liveness-detection "Liveness Detection")
+
+### Face Detection
+Detect faces and analyze them from an binary data. The latest model is the most accurate and recommended to be used.
+For the detailed differences between different versions of **Detection** and **Recognition** model, please refer to the following links.
+* [Detection model](evaluate_different_detection_models)
```

Comment:
There are some broken link including but not limited to this one.

Reply:
Fixed them at commit: 0309e20e50e376e577972c0ca56e0a645a7ec42a.